packages/peer-exchange/src/waku_peer_exchange_discovery.ts

```diff
@@ -109,6 +126,36 @@ export class PeerExchangeDiscovery
     return "@waku/peer-exchange";
   }
 
+  private readonly startRecurringQueries = async (
+    peerId: PeerId
+  ): Promise<void> => {
+    const peerIdStr = peerId.toString();
+    const {
+      initialQueryTimeout = DEFAULT_PEER_EXCHANGE_INITIAL_QUERY_TIMEOUT_MS,
+      queryInterval = DEFAULT_PEER_EXCHANGE_QUERY_INTERVAL_MS,
+      attemptsBeforeAbort = DEFAULT_ATTEMPTS_BEFORE_ABORT,
+    } = this.options;
+
+    setTimeout(async () => {
+      await this.query(peerId);
+    }, initialQueryTimeout);
```

Comment:
I thought we already discussed that this is not needed as the recurring queries should start as soon as we are connected to the peer. Sorry. can't find my initial comment.